addons/mail/models/mail_thread.py

```diff
@@ -2114,7 +2114,7 @@ def _notify_record_by_inbox(self, message, recipients_data, msg_vals=False, **kw
             } for pid in inbox_pids]
             self.env['mail.notification'].sudo().create(notif_create_values)
 
-            message_format_values = message.message_format()[0]
+            message_format_values = message.message_format(fetch_followers=True)[0]
```

Comment:
I'm not sure if this is a good idea to add the info here, and this also is why we have extra queries.

This bus notif is useful only as a quick notification to alert the user of new notifications. The full fetch should probably come when the user is visiting inbox.

But the main issue I have here is security again, because this message is sent on the bus, it is sent with the access right of the user generating the message/notification, and not with the access right of the user receiving it.

Reply:
This comment is still unanswered/unresolved.